Dockerfile.acceptance-testing

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.4
+
+EXPOSE 8080
+
+COPY ./inventory /usr/bin/
+COPY ./config.yaml /usr/bin/
+
```

Comment:
BTW. i'm wondering, if we already tag an image as mendersoftware/inventory:pr you could use that image as `FROM`